diff -burN webgrep-2.11.orig/Makefile webgrep-2.11/Makefile
--- webgrep-2.11.orig/Makefile	Mon Jan  6 23:13:20 2003
+++ webgrep-2.11/Makefile	Tue Jul 22 14:39:22 2003
@@ -1,7 +1,7 @@
 # written by Guido Socher 
 PREFIX=$(DESTDIR)/usr
-MANDIR=$(PREFIX)/man
+MANDIR=$(PREFIX)/share/man
 INSTALL=install
 MANP=man1/srcgrep.1 man1/hrefgrep.1 man1/blnkcheck.1 man1/lshtmlref.1 man1/httpcheck.1 man1/webfgrep.1 man1/taggrep.1
 #-----------
